connector_helpscout/models/helpscout_conversation/common.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 LasLabs Inc.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import models, fields
+from odoo.addons.component.core import Component
+
+
+class ProjectTask(models.Model):
+
+    _inherit = 'project.task'
+
+    helpscout_bind_ids = fields.One2many(
+        string='HelpScout Bindings',
+        comodel_name='helpscout.conversation',
+        inverse_name='odoo_id',
+    )
+    helpscout_folder_id = fields.Many2one(
```

Comment:
Hmmm typically we want all helpscout context in the bindings, vs having them in the concrete models. Is there a reason for this?